src/home_work3/models/UserModel.js

```diff
@@ -0,0 +1,83 @@
+const uuid = require("uuid/v4");
+let Sequelize = require("sequelize");
+let dbConnector = require("../db/dbConnector");
+
+var User = dbConnector.define(
```

Comment:
Consider from now on using only "let" or "const" since they provide the same functionality plus additional features.

Reply:
Resolved as per the comment